systemd/xmtpd-prune.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=xmtpd-prune
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+EnvironmentFile=/etc/xmtpd.env
+ExecStart=/usr/local/bin/xmtpd-prune
```

Comment:
pruner is intended to be a batch/CRON job. Are you running it continually?

Reply:
ours is running nightly, but weekly is currently fine given the low load on testnet.